backend/src/schema/resolvers/users.spec.js

```diff
@@ -119,12 +108,12 @@ describe('users', () => {
         await expect(client.request(mutation, variables)).resolves.toEqual(expected)
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-03T18:39:04Z" title="Wednesday, July 3rd 2019, 8:39:04 pm +02:00">Jul 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -119,12 +100,12 @@ describe('users', () => {
         await expect(client.request(mutation, variables)).resolves.toEqual(expected)
```


don't know what this is testing...

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-03T20:16:27Z" title="Wednesday, July 3rd 2019, 10:16:27 pm +02:00">Jul 3, 2019</time>_

---


I think it is testing that you cannot alter other user accounts, only your own.